peerconnection.go

```diff
@@ -641,6 +589,12 @@ func (pc *PeerConnection) addAnswerMediaTransceivers(d *sdp.SessionDescription)
 		return nil, err
 	}
 
+	if !pc.iceGatherer.agentIsTrickle {
+		if err = pc.iceGatherer.Gather(); err != nil {
```

Comment:
Are you should it should start gathering here? It used to start gathering during initialization before.

Reply:
We can't do it during init because people need time to set the `OnICECandidate` callback.

Yea changing what function blocks isn't great, but I don't think users will notice in the normal case.

Reply:
ahhhh... makes total sense. In my local version I really struggled with that.